src/proc.cpp

```diff
@@ -515,6 +515,8 @@ static int process_clean_after_marking(bool allow_interactive) {
 
             // TODO: The generic process-exit event is useless and unused.
             // Remove this in future.
+            // Update: This event is used for cleaning up the psub temporary files and folders.
```

Comment:
Huh? Is this the same event as the "--on-job-exit caller" event?

Reply:
I know, I was confused too. `PROCESS_EXIT` is fired from two places in the codebase and I can't see where it's used. This PR is already getting out of hand so I don't want to figure out why it breaks the job-exit event/the psub cleanup, but it does.

Reply:
Oh, yeah, sure, by all means keep this as-is!